app/assets/stylesheets/touchpoints-widget.css

```diff
@@ -0,0 +1,15 @@
+/*
+ * This is a manifest file that'll be compiled into application.css, which will include all the files
```

Comment:
is it possible to include this CSS in the `widget.css.erb` file?

Combining the CSS would keep the same number of calls from a client's browser to Touchpoints, without adding an extra call for .css in this case.